cypress/support/commands.js

```diff
@@ -23,3 +23,6 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+ Cypress.Commands.add('findByPlaceholder', (placeholder) => {
```

Comment:
```suggestion
 Cypress.Commands.add('findByPlaceholder', (placeholder) => {
```
you do not use this function. Remove it